configs/templates/MetricTemplate.proto

```diff
@@ -8,62 +8,7 @@ import "mixer/configs/mixer/TemplateExtensions.proto";
 option (istio.mixer.v1.config.template.template_variety) = TEMPLATE_VARIETY_REPORT;
 option (istio.mixer.v1.config.template.template_name) = "Metric";
 
-message Type {
+message Constructor {
```

Comment:
This should still be a `Type` or a `Template`, right? From that we generate the `Constructor`, which swaps fields from `ValueType` to `string`.